src/style.css

```diff
@@ -0,0 +1,77 @@
+* {
+  box-sizing: border-box;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 400;
+  font-style: normal;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  font-style: normal;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.search {
+  position: relative;
+}
+
+.search__ipt {
+  display: block;
+  width: 100%;
+  padding: 20px 40px 20px 62px;
+  height: 70px;
+  font-family: Avenir, sans-serif;
```

Comment:
move it to `html` tag selector and let inputs inherit it

Reply:
I didn't quite understand what to do. I moved some styles to " * " for inherit inputs

Reply:
> move it to `html` tag selector and let inputs inherit it

I didn't quite understand what to do. I moved some styles to " * " for inherit inputs